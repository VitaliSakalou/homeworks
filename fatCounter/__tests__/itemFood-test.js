"use strict";

import React from 'react';
import renderer from 'react-test-renderer';

import ItemFood from '../components/ListOfFood/itemFood/itemFood';


test('работа ItemFood ', () => {

  const component = renderer.create(
    <ItemFood/>
  );

  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  // componentTree.props.onClick();

  // componentTree=component.toJSON();
  // expect(componentTree).toMatchSnapshot();
    
});
