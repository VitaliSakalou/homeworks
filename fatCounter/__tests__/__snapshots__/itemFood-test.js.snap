// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа ItemFood  1`] = `
<li
  className="item undefined"
  data-identifier={undefined}
  onClick={[Function]}
>
  <span
    className="btnundefined"
  >
    <button
      className="btnBody"
      data-identifier={undefined}
      onClick={[Function]}
    >
      -
    </button>
    <span
      className="btnCount"
    />
    <button
      className="btnBody"
      data-identifier={undefined}
      onClick={[Function]}
    >
      +
    </button>
  </span>
</li>
`;
